tests/onnxruntime/test_modeling.py

```diff
@@ -1277,6 +1278,117 @@ def test_compare_to_io_binding(self, *args, **kwargs):
         gc.collect()
 
 
+class ORTModelForImageSegmentationIntegrationTest(unittest.TestCase):
```

Comment:
It seems to be similar to `ORTModelForImageClassification`, maybe we could abstract all of these test with a base test class, can be done in another PR